server/parsec/components/memory/account.py

```diff
@@ -17,7 +23,11 @@ async def create(self, email: str) -> None | CreateAccountBadOutcome:
         if email in self._data.accounts:
             return CreateAccountBadOutcome.ACCOUNT_ALREADY_EXISTS
         else:
-            self._data.accounts[email] = MemoryAccount(email)
+            try:
+                user_email = UserAccountDataValidationModel(user_email=email)
+                self._data.accounts[email] = MemoryAccount(user_email=user_email.user_email)
```

Comment:
@FirelightFlagboy I'm not exactly sure that's the canonical way to use pydantic for data validation